jira/client.py

```diff
@@ -3983,6 +4041,16 @@ def permissionschemes(self):
 
         return data["permissionSchemes"]
 
+    @lru_cache(maxsize=None)
+    def issuetypeschemes(self):
+
```

Comment:
```suggestion
    def issue_type_schemes(self) -> List[IssueTypeScheme]:
    """<docstring>"""
```

Just a reminder to use snake case for function names, to add the right typehints and to add a sensible docstring.

Reply:
Reminder: Docstrings should conform to the Google docstring style and include return type, I can still see some rogue commas in the return sections too